.vscode/aiven.code-snippets

```diff
@@ -91,5 +91,11 @@
       "</Tabs>",
       "$5"
     ]
+  },
+  "Prerequisites": {
+    "scope": "markdown",
+    "prefix": "pre",
+    "body": ["## Prerequisites", "", "- Access to the [Aiven Console](https://console.aiven.io/)", "- "],
```

Comment:
You can use a dollar sign to move the cursor to a specific position ✨ :
```suggestion
    "body": ["## Prerequisites", "", "- Access to the [Aiven Console](https://console.aiven.io/)", "- $1"],
```

Can you also check that you've re-enabled Prettier and that it formats this file when you save?